### DISCUSSION TITLE
Google Search Api not Working

### DISCUSSION THREAD

**Discussion by dsnetgo** [langgenius/dify#10318] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

Hi. Google Search Api not Working. 

Can you fix this?
On version 9.2 everything works.
It doesn't work on version 10.x, 11.x 
![bbb1](https://github.com/user-attachments/assets/8687152d-9649-4b5f-8f10-562f74c1adb9)

**Comment by Vampxgg** [comment]
我也出现了(psycopg2.errors.InvalidTextRepresentation) invalid input syntax for type uuid: "2149544) " LINE 3: WHERE tool_files.id = '2149544) ^ [SQL: SELECT tool_files.id, tool_files.user_id, tool_files.tenant_id, tool_files.conversation_id, tool_files.file_key, tool_files.mimetype, tool_files.original_url, tool_files.name, tool_files.size FROM tool_files WHERE tool_files.id = %(id_1)s::UUID] [parameters: {'id_1': '2149544)\n'}] (Background on this error at: https://sqlalche.me/e/20/9h9h)
